README.md

```diff
@@ -1,92 +1,224 @@
-|Intel(R) DAAL|2020 Gold|[rls/daal-2020-rls](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-rls)</br>[rls/daal-2020-mnt](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-mnt) (contains ongoing fixes)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/en-us/daal)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Developer Guide](https://software.intel.com/en-us/daal-programming-guide)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/en-us/articles/intel-data-analytics-acceleration-library-2020-system-requirements)|
+Intel&reg; oneDAL K-means fit, strong scaling result | Intel&reg; oneDAL K-means fit, weak scaling results
+:-------------------------:|:-------------------------:
+![](docs/readme-charts/Intel%20oneDAL%20KMeans%20strong%20scaling.png)  | ![](docs/readme-charts/intel%20oneDAL%20KMeans%20weak%20scaling.png)
 
 ## Installation
 
-You can install oneDAL: 
+You can install oneDAL:
 
-- from [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) as a part of Intel(R) oneAPI Base Toolkit.
+- from [oneDAL home page](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onedal.html) as a part of Intel&reg; oneAPI Base Toolkit.
 - from [GitHub\*](https://github.com/oneapi-src/oneDAL/releases).
 
+## Installation from Source
 See [Installation from Sources](INSTALL.md) for details.
 
-## Examples
+## Technical Preview Features
+
+Technical preview features are introduced to gain early feedback from developers. A technical preview feature is subject to change in the future releases. Using a technical preview feature in a production code base is therefore strongly discouraged.
+In C++ APIs, technical preview features are located in `daal::preview` and `onedal::preview` namespaces. In Java APIs, technical preview features are located in packages that have the `com.intel.daal.preview` name prefix.
+The only preview feature at the moment is `MultiNodeBatch` for K-Means, a stepless distributed algorithm based on oneCCL.
+
+## Python API
+
+Intel&reg; oneDAL has a python API that is provided as a standalone python library called [daal4py](https://github.com/IntelPython/daal4py).
+Below is an example of how daal4py can be used for calculation KMeans clusters
+
+```python
+import numpy as np
+import pandas as pd
+import daal4py as d4p
+
+data = pd.read_csv("local_kmeans_data.csv", dtype = np.float32)
+
+init_alg = d4p.kmeans_init(nClusters = 10,
+                           fptype = "float",
+                           method = "randomDense")
+
+centroids = init_alg.compute(data).centroids
+alg = d4p.kmeans(nClusters = 10, maxIterations = 50, fptype = "float",
+                 accuracyThreshold = 0, assignFlag = False)
+result = alg.compute(data, centroids)
+```
+
+Often data scientists require different tools for analysis regular and big data. daal4py offers various processing models, which makes it easy to enable distributed multi-node mode.
+```python
```

Comment:
Think it worth it to add examples with GPU support and DPPL - @rlnx  @michael-smirnov   - any candidates for adding here?

Reply:
Good idea. It could be added as a new paragraph in Python API section